docker/clickhouse/options_method_test.xml

```diff
@@ -0,0 +1,25 @@
+<clickhouse>
+    <http_options_response>
+        <header>
+            <name>Access-Control-Allow-Origin</name>
+            <value>http://grafana:3000</value>
```

Comment:
let's just change to `*` and use `options_method.xml`

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSNotSupportingCredentials?utm_source=devtools&utm_medium=firefox-cors-errors&utm_campaign=default
![Screenshot from 2024-06-27 14-41-22](https://github.com/Altinity/clickhouse-grafana/assets/44409195/70ab2000-8fd1-4d6f-bec2-679af5f1b1b2)